src/renderer/components/_molecules/variables-table/editable-cell.tsx

```
@@ -340,6 +340,321 @@ const EditableNameCell = ({
+        />
+      ) : (
+        <div
+          onClick={handleStartEditing}
+          className={cn('flex w-full flex-1 bg-transparent p-2 text-center outline-none', {
+            'pointer-events-none': !selected,
+            'cursor-not-allowed': !isEditable(),
+          })}
+        >
+          <p
+            className={cn('h-4 w-full max-w-[400px] overflow-hidden text-ellipsis break-all', {})}
+            dangerouslySetInnerHTML={{ __html: formattedCellValue }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)